python/tk_multi_workfiles/entity_tree/entity_tree_form.py

```diff
@@ -117,7 +121,7 @@ def __init__(self, entity_model, search_label, allow_task_creation, extra_fields
                 self._ui.entity_tree.setModel(filter_model)
 
                 # connect up the filter controls:
-                self._ui.search_ctrl.search_edited.connect(self._on_search_changed)
+                self._ui.search_ctrl.search_changed.connect(self._on_search_changed)
```

Comment:
I've added a tooltip, which isn't a bad idea, but I don't think everyone will think to check that when they realise it doesn't search automatically. I can't really think of a great solution.

We could have a setting in the config, but that really does feel a bit obscure and over the top.